ko/sql/schema/server_stmt.rst

```diff
@@ -0,0 +1,378 @@
+  
+.. code-block:: sql
+
+    CREATE SERVER dblink_srv1 (
+	   HOST='192.168.1.8',
+	   PORT=3300,
+	   DBNAME=demodb,
+	   USER=dba,
+	   PASSWORD='password1234',
+	   PROPERTIES='?rcTime=600',
+	   COMMENT='this is dblink_srv1'	   
+    );
+
+다음은 서버 생성 시 최소한의 정보만 포함하는 예제이다.
+원격지의 demodb에 비밀번호가 없는 dev1 계정으로 접속 할 것임을 나타내고 있다. 
+srv1, srv2, srv3는 동일한 의미이다.
+
+.. code-block:: sql
+
+    CREATE SERVER srv1 (
+	   HOST='192.168.1.8',
+	   PORT=3300,
+	   DBNAME=demodb,
+	   USER=dev1	 
+    );
+    
+   CREATE SERVER srv2 (
+	   HOST='192.168.1.8',
+	   PORT=3300,
+	   DBNAME=demodb,
+	   USER=dev1,
+	   PASSWORD=       	 
+    );
+    
+    CREATE SERVER srv3 (
+	   HOST='192.168.1.8',
+	   PORT=3300,
+	   DBNAME=demodb,
+	   USER=dev1,
+	   PASSWORD=''       	 
+    );
+    
+다음은 서버 생성 시 소유자를 지정하는 예제이다.
+소유자 지정없이 CREATE하는 경우에는 현재 사용자가 소유자가 된다.
+추후 ALTER SERVER 구문을 이용해서 소유자를 변경할 수 있다.
+아래 예제에서 두 서버의 이름이 동일하게 *srv2* 로 같지만 소유자는 각각 *dba* 와 *cub* 로 다르다.
+
+.. code-block:: sql
+
+    -- When the current account is dba
+    CREATE SERVER srv2 (
+	   HOST='192.168.1.8',
+	   PORT=3300,
+	   DBNAME=demodb,
+	   USER=dev1,
+	   PASSWORD='dev1-password',
+	   COMMENT='The owner of this server is dba'
+    );
+    
+   CREATE SERVER cub.srv2 (
+	   HOST='192.168.1.8',
+	   PORT=3300,
+	   DBNAME=demodb,
+	   USER=dev2,
+	   PASSWORD='dev2-password',
+	   COMMENT='The owner of this server is cub.'
+    );
+
+.. code-block:: sql
+    
+    CREATE SERVER srv1 ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER "srv 1" ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER "srv.1" ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER cub.srv1 ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    
+    CREATE SERVER "cub"."srv 2" ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+    CREATE SERVER [cub].[srv.2] ( HOST='localhost', PORT=3300, DBNAME=demodb, USER=dev1 );
+        
+
+위의 예시에서 *"srv.1"* 과 *[cub].[srv.2]*  이름으로 생성은 실패한다.
+*cub.srv1* 의 경우 *cub* 는 사용자 이름으로 인식되고 서버명은 *srv1* 으로 인식 될 것이다.
+        
+.. note::
+
+    서버명에는 점('.')을 사용 할 수 없습니다. 따옴표나 []로 묶은 경우라고 해도 서버명에 점('.')을 사용할 수는 없습니다.
+
+
+
+RENAME SERVER
+=============
+
+**RENAME SERVER** 구문을 사용하여 서버 이름을 변경할 수 있다.
+
+::
+
+    RENAME SERVER [owner_name.] old_server_name AS new_server_name ;
+            
+        
+*   *owner_name*: 이름을 변경할 대상 서버의 소유자 이름을 지정한다.
+*   *old_server_name*: 이름을 변경할 대상 서버의 이름을 지정한다.
+*   *new_server_name*: 새롭게 부여될 서버의 이름을 지정한다(최대 254바이트).
+
+.. note::
+
+    서버의 소유자 또는 소유 그룹의 멤버만 정보를 변경 할 수 있다. 
+    특별히 **DBA** 또는 **DBA** 의 멤버는 모든 서버 정보를 변경할 수 있다.
+    
+    변경 후에도 소유자는 동일하게 유지된다. 소유자를 변경하기 위해서는 :ref:`ALTER SERVER 구문<owner_to>`\ 을 참고한다.
+
+
+.. code-block:: sql
+
+    -- When the current account is dba
+    RENAME SERVER srv1 AS srv2;
+    RENAME SERVER dev1.srv1 AS srv3;
+
+..
+
+위 예제가 *dba* 계정에서 수행 되었다면 *srv1* 서버의 소유자가 *cub* 였다면 변경된 *srv2* 의 소유자도 *cub* 이다.

```

Comment:
ambiguous sentence. and It is recommended to use the active voice.

```suggestion
위 예제를 *dba* 계정에서 수행하더라도, *srv2* 의 소유자는 변경되지 않고 *srv1* 서버의 소유자인 *cub* 로 유지한다.
```

Reply:
It's a clear, unambiguous phrase. Thanks a lot.